core/os/os.cpp

```diff
@@ -373,8 +373,16 @@ OS::ScreenOrientation OS::get_screen_orientation() const {
 void OS::_ensure_data_dir() {
 
 	String dd = get_data_dir();
-	DirAccess *da = DirAccess::open(dd);
+	String current_dir;
```

Comment:
That's good now, thanks. Could you also squash the two commits together? See http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase